src/applications/check-in/package.json

```diff
@@ -6,7 +6,7 @@
     },
     "licence": "MIT",
     "dependencies": {
-        "@datadog/browser-rum": "^4.22.0",
+        "@datadog/browser-rum": "^4.45.0",
```

Comment:
This should probably get moved from the check-in workspace to the main package.json since it's getting used by multiple applications now. Now is probably as good a time as any, what do you think @department-of-veterans-affairs/patient-check-in ?

Also it would make sense to me to do that work in a separate PR without the 526EZ changes...

Reply:
that all makes sense to me

Reply:
I'll look into opening a PR during work hours for the version change.  For sure it's strange that's where the package was...

Reply:
It was originally added to the check-in workspace because that was the only app using RUM initially.